src/atoms/lp_cone/min.jl

```diff
@@ -78,4 +78,4 @@ end
 min(x::AbstractExpr, y::AbstractExpr) = MinAtom(x, y)
 min(x::AbstractExpr, y::Value) = min(x, Constant(y))
 min(x::Value, y::AbstractExpr) = min(Constant(x), y)
-neg(x::AbstractExpr) = min(x, Constant(0, Negative()))
+neg(x::AbstractExpr) = max(-x, 0)
```

Comment:
```suggestion
neg(x::AbstractExpr) = max(-x, Constant(0, Negative()))
```
This ensures that the result is `Negative()` even if the maximum is 0. IIRC, 0 is treated as `Positive()`, which is likely why Convex thinks it's not DCP-compliant.

Reply:
Comparing the two different cases at the command line yielded no difference. Despite that fact, for the sake of pedantry, I believe Negative() should now be Positive() given the change since `neg` returns positive values (-x negates x, and the max returns the new positive entries.)

It is also possible to define this function as `pos(-x)`, which also rightly returns `Positive()`.

Reply:
I agree, I think it should be `Positive` now

Reply:
Is it important to enforce `Positive()`, or is the `0` literal positive by default?

Reply:
Doesn't hurt to be explicit, so I think `Constant(0, Positive())` would be reasonable here.